hr_expense_substate/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+#. Go to ** Settings > Technical > Database Structure ** and Add  "Base substate".
+    If necessery you can add "target State values" (ex define a substate for "cancel"
+    state).
```

Comment:
```suggestion
#. Go to ** Settings > Technical > Substate Configuration > Base Substate ** and add a "Base Substate".
    If necessary you can add "target State values" (e.g. define a substate for "cancel"
    state).
```

Nitpicking as discovered in: https://github.com/OCA/account-invoicing/pull/1595#discussion_r1669815229